README.rst

```diff
@@ -267,9 +267,11 @@ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 .. |badge_license| image:: https://img.shields.io/pypi/l/pypsa.svg
     :target: License
 
-.. |badge_travis| image:: https://img.shields.io/travis/PyPSA/PyPSA/master.svg
-    :target: https://travis-ci.org/PyPSA/PyPSA
-    :alt: Build status on Linux
+.. |ci_badge_conda| image:: https://github.com/pypsa/pypsa/actions/workflows/CI-conda.yml/badge.svg
+    :target: https://github.com/pypsa/pypsa/actions/workflows/CI-conda.yml
+
+.. |ci_badge| image:: https://github.com/pypsa/pypsa/actions/workflows/CI.yml/badge.svg
```

Comment:
Atm the badge shows an unknown value. Does that change as soon as we have pulled into the master or is that unintended?

Reply:
You mean the codecov below? Yes, it'll show the coverage of the master which is only built when it's first merged.

Reply:
alright, then let's pull it in then